### PR TITLE
Fixes #1769: MessageComposer.react.js refactored

### LINKED ISSUE #1769: Refactor MessageComposer.react.js
**Actual Behaviour**

**Expected Behaviour**

**Steps to reproduce it**

**LogCat for the issue**

**Screenshots of the issue**

**Would you like to work on the issue?**

### PR DESCRIPTION
**Pull request by AakashMallik**
Fixes #1769 

Changes: The changes I propose are:
- [x] Grouping the import statement in a logical fashion. 
- [x] Replacing let with const wherever possible.
- [x] Using ES6 string templates instead of concatenation.
- [x] Simplify function logic and minor optimizations.

Demo Link: https://pr-1770-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/ChatApp/MessageComposer.react.js [hunk 1]
```diff
@@ -151,103 +220,121 @@ class MessageComposer extends Component {
...
-    }
-
-    let testSkill = $.urlParam('testExample');
-    if (testSkill) {
-      let text = testSkill.trim();
+      let text = text.trim();
       if (text) {
-        let enterAsSend = UserPreferencesStore.getEnterAsSend();
-        if (!enterAsSend) {
+        let EnterAsSend = UserPreferencesStore.getEnterAsSend();
```
Comment (samagragupta) [inline src/components/ChatApp/MessageComposer.react.js]: I think you should let the first letter small.
`EnterAsSend` to `enterAsSend`

Comment (AakashMallik) [reply src/components/ChatApp/MessageComposer.react.js]: @samagragupta Those functions have been defined in a different component, so I thought I should not mess with their names. I know camel case should be used but seems like the previous devs forgot to use it.

Comment (akshatnitd) [reply src/components/ChatApp/MessageComposer.react.js]: Make it camelCase. The correction needs to made in other files while refactoring them.

**Comment by AakashMallik** [comment]
@akshatnitd Sir, I have made the changes you asked for. :innocent:

**Pull request closed without merging**